### DISCUSSION TITLE
Can I add a new font to pyreportjasper?

### DISCUSSION THREAD

**Discussion by hoangson0523** [acesseonline/pyreportjasper#88] [Q&A]
My team decided to use pyreportjasper in our python project. We have some report with Thai language and we have to use a new font. I try to add a font jar to PyReportJasper config but Thai character not showing up. How can I add a new font to pyreportjasper?

**Comment by jadsonbr** [comment]
First step is to check if it is possible to install the font in the place where the library will run, if this is possible, no other step is necessary besides defining the font in the construction of the report. If the source is not available for installation, some more complex steps will be necessary. Below are the necessary procedures to be performed.


Create a font extension in Jasper Studio. [Click here and see how to do it.](https://community.jaspersoft.com/documentation/tibco-jaspersoft-studio-user-guide/v640/fonts)

Export this extension. You will get .jar for this export. And after that add it as a resource in pyreportjasper.

Even in JasperStudio if you are exporting to PDF and do not have this extension it will not work. Only when previewing within JasperStudio will you get the expected result. For the correct export it is necessary to add the font.

**Reply by gustavo-hm1** [reply]
I tried to add the helvetica but it does not work, looks like the jvm cannot identify or is no available, can you help me?
